src/test/java/marquez/AbstractIntegrationTest.java

```diff
@@ -0,0 +1,38 @@
+package marquez;
+
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import java.net.URL;
+import marquez.client.MarquezClient;
+import marquez.client.Utils;
+import org.jdbi.v3.testing.JdbiRule;
+import org.junit.ClassRule;
+
+public abstract class AbstractIntegrationTest {
```

Comment:
Nice! Great idea to move common test setup to a base class other test suites can extend. Mind renaming the class to `BaseIntegrationTest` to best convey it's usage?